src/main/app/src/app/taken/taak-view/taak-view.component.ts

```diff
@@ -571,4 +585,29 @@ export class TaakViewComponent
       zaaktype,
     } satisfies Partial<GeneratedType<"RestZaak">> as GeneratedType<"RestZaak">;
   }
+
+  onDocumentCreate(event: FormioCustomEvent) {
+    this.activeSideAction = "actie.document.maken";
+    this.smartDocumentsGroupPath = this.formioGetSmartDocumentsGroups(
+      event.component,
+    );
+    const componentBaseName = event.component.key
```

Comment:
This splitting and slicing seems a bit magical to me. Can we not add a code comment explaining why this is done?